bring/src/test/java/com/bobocode/petros/bring/scanner/mocks/EveningBeanPostProcessor.java

```diff
@@ -0,0 +1,21 @@
+package com.bobocode.petros.bring.scanner.mocks;
+
+import com.bobocode.petros.bring.context.domain.BeanReference;
+import com.bobocode.petros.bring.factory.postprocessor.BeanPostProcessor;
+
+public class EveningBeanPostProcessor implements BeanPostProcessor {
```

Comment:
May I ask you to move all there mock classes you've prepared for testing to a package -> postprocessor? So your package structure will look like this: "com.bobocode.petros.bring.scanner.mocks.postprocessor". This way your mock classes are stored separately and it won't be mixed with other classes in case anyone will add something else in a  separate PR.